BlackNectar/BlackNectar/StoresDataStruct.swift

```diff
@@ -16,27 +16,45 @@ struct StoresInfo {
     let storeName: String
     let location: NSDictionary
     let address: NSDictionary
-    let storeImage: String
+    let storeImage: URL
+    
     
     static func fromJson(dictionary: NSDictionary) -> StoresInfo? {
         
         guard let storeName = dictionary ["store_name"] as? String,
             let location = dictionary ["location"] as? NSDictionary,
-            let address = dictionary ["address"] as? NSDictionary else {
+            let address = dictionary ["address"] as? NSDictionary,
+            let storeType = dictionary ["main_image_url"] as? String,
```

Comment:
Might want to put these keys into a separate class as permanent static variables